src/components/templates/PartyMap/components/PartyMapPreview/PartyMapPreview.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react";
+import { PartyMapRoomData } from "types/PartyMapRoomData";
+
+interface PartyMapPreviewProps {
+  expanded: boolean;
+  backgroundImage?: string;
+  backgroundRatio: number;
+  avatarPosition: {
+    x: number;
+    y: number;
+  };
+  rooms: PartyMapRoomData[];
+  onIconClick: () => void;
+}
+
+const PREVIEW_WIDTH_PIXELS = 120;
+const PREVIEW_WIDTH_VH = 80;
+
+export const PartyMapPreview: FC<PartyMapPreviewProps> = ({
```

Comment:
[nit] I usually prefer to use `React.FC` here for better context, rather than importing it individually. `FC` on it's own doesn't provide as much immediate/meaningful context as using the full `React.FC` in my opinion.